client/src/components/AnimeList/AnimeList.js

```diff
@@ -1,18 +1,34 @@
-import React from "react";
+import React from 'react'
+const AnimeList = ({ animelist,setAnimeInfo,animeComponent,handleList}) => {
+  const AddToList=animeComponent;
+  return (
+    <>
+      {
+        animelist ? (
+          animelist.map((anime, index) => {
+            return (
+              <div className="card" key={index} onClick={()=>setAnimeInfo(anime)}>
```

Comment:
It would probably make sense to create a separate `AnimeCard` component to put this logic in there and then make the `AnimeList` component a map creating a list of `AnimeCard` components.